ai_worker/src/user_rag/hybrid_search.py

```
@@ -0,0 +1,174 @@
+
+    기능:
+    - 케이스 증거 검색 (SearchEngine)
+    - 법률 지식 검색 (LegalSearchEngine)
+    - 두 결과를 통합하여 반환
+    - 거리 기준 정렬
+    """
+
+    def __init__(
+        self,
+        storage_manager=None,
+        evidence_collection: str = "leh_evidence",
```

Ruff (0.14.5)

Unused method argument: `evidence_collection`

(ARG002)